bundles/org.openhab.binding.modbus/src/main/feature/feature.xml

```diff
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <features name="org.openhab.binding.modbus-${project.version}" xmlns="http://karaf.apache.org/xmlns/features/v1.4.0">
-    <repository>file:${basedirRoot}/bundles/org.openhab.io.transport.modbus/target/feature/feature.xml</repository>
+    <repository>mvn:org.openhab.core.features.karaf/org.openhab.core.features.karaf.openhab-core/${project.version}/xml/features</repository>
+    <repository>mvn:org.openhab.addons.bundles/org.openhab.io.transport.modbus/${project.version}/xml/features</repository>
```

Comment:
The problem with this approach is: If the transport bundle has not been compiled yet (which is the only reason I can image why the `file`-repo fails), an old version is used. That could be wrong.